yarn.lock

```diff
@@ -1776,7 +1776,7 @@
     "@jridgewell/gen-mapping" "^0.3.0"
     "@jridgewell/trace-mapping" "^0.3.9"
 
-"@jridgewell/sourcemap-codec@1.4.14", "@jridgewell/sourcemap-codec@^1.4.10":
+"@jridgewell/sourcemap-codec@^1.4.10", "@jridgewell/sourcemap-codec@1.4.14":
```

Comment:
The changes required should not have any affiliations with the yarn.lock file.

Reply:
@palisadoes should the yarn.lock file be pushed along the changes. Additionally, @sarazaToussaint you can remove the yarn.lock in place of package-lock.json.

Reply:
@Rumone 

```yarn.lock``` file must not be pushed. Moreover @sarazaToussaint How have you pushed it, it was included in ```.gitignore``` file.

Reply:
@anshgoyalevil, @Rumone, @Kevoniat, @ttaylor92 I resolve the conflicts and made all changes required, I hope this PR is ready to be merged.